docs/core/diagnostics/FAQ-dumps.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Dumps - .NET FAQ
+description: Frequently Asked Questions about dumps .NET
+ms.date: 09/19/2022
+---
+
+# Frequently Asked Questions Regarding Dumps
+
+## Collecting dumps on macOS and Linux
+
+Collecting a dump requires that the processes in the system have access to call `ptrace` targeting parent processes. The following is a non-exhaustive list of settings that are needed for this to work properly:
+
+- On Linux-based systems, `/proc/sys/kernel` should have `ptrace_scope` set to `1` or `0`. Any value higher than this will require running as root or will block dump collection entirely.
```

Comment:
```suggestion
- On Linux-based systems, set `ptrace_scope` for `/proc/sys/kernel` to `1` or `0`. Any value higher than this requires running as root to prevent dump collection from being blocked.
```